docs/index.rst

```diff
@@ -5,13 +5,98 @@
 
 Welcome to nanopub's documentation!
 ===================================
-Documentation for nanopub is still work in progress
+The ``nanopub`` library provides a high-level, user-friendly python
+interface for searching, publishing and retracting nanopublications.
+Nanopublications are FAIR data containers for scientific results
```

Comment:
```suggestion
Nanopublications are a formalized and machine-readable way of communicating the smallest possible units of publishable information. This could be, for example, the outcome of a scientific study or a claim made by a particular scientist. Nanopublications are searchable, citable, and contain authorship and attribution information. The aim is to encourage individual scientific results to be released in a traceable and interoperable format. As such, nanopublications are an effective `FAIR <https://www.go-fair.org/fair-principles/>`_ means of communicating scientific claims and results. Read more about them at `http://nanopub.org/ <http://nanopub.org/>`_.
```

Reply:
I have taken a few bits and pieces so the explanation is a tiny bit more detailed on the landing page. I've added the detailed description to the section 'What are nanopublications?'